backend/dataall/core/permissions/api/resolvers.py

```diff
@@ -3,40 +3,29 @@
 
 from dataall.base.aws.sts import SessionHelper
 from dataall.base.aws.parameter_store import ParameterStoreManager
-from dataall.core.permissions.db.tenant_policy_repositories import TenantPolicy
+from dataall.base.db.exceptions import RequiredParameter
+from dataall.core.permissions.services.tenant_policy_service import TenantPolicyService
 
 log = logging.getLogger(__name__)
 
 
 def update_group_permissions(context, source, input=None):
-    with context.engine.scoped_session() as session:
-        return TenantPolicy.update_group_permissions(
-            session=session,
-            username=context.username,
-            groups=context.groups,
-            uri=input['groupUri'],
-            data=input,
-            check_perm=True,
-        )
+    if not input['groupUri']:
+        raise RequiredParameter('groupUri')
+    return TenantPolicyService.update_group_permissions(
+        data=input,
+        check_perm=True,
+    )
 
 
 def list_tenant_permissions(context, source):
-    with context.engine.scoped_session() as session:
-        return TenantPolicy.list_tenant_permissions(session=session, username=context.username, groups=context.groups)
+    return TenantPolicyService.list_tenant_permissions()
 
 
 def list_tenant_groups(context, source, filter=None):
-    if not filter:
+    if filter is None:
```

Comment:
`not filter` will evaluate all the [falsey values](https://www.geeksforgeeks.org/truthy-vs-falsy-values-in-python/#:~:text=Falsy%20Values%20Includes%3A) to true, hence this is not functionally equivalent. Do you see this as a problem here?